fedimint-wasm-tests/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 #![deny(clippy::pedantic)]
 #![allow(clippy::large_futures)]
 #![allow(dead_code)]
+#![allow(clippy::literal_string_with_formatting_args)]
```

Comment:
I had to deactivate the lint here due to CI failures that don't make any sense. https://github.com/fedimint/fedimint/actions/runs/14813184565/job/41590426772?pr=7343#step:7:123

My guess is that some WASM-related macro has a problem that gets detected by clippy and then clippy doesn't know where it originated from.